<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpUdpPort</class>
 <widget class="QWidget" name="TcpUdpPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>20</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="protocolBox">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox { width: 90px }
</string>
     </property>
     <item>
      <property name="text">
       <string notr="true">TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">UDP</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ipLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ipEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit { width: 7em }</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="portEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit { width: 4em }</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
